openvdb_points_houdini/houdini/SOP_OpenVDB_Points.cc

```diff
@@ -108,7 +107,7 @@ convertAttributeFromHoudini(PointDataTree& tree, const PointIndexTree& indexTree
         defaultValue = TypedMetadata<ValueType>(value).copy();
     }
 
-    appendAttribute<AttributeType, PointDataTree>(tree, name, stride, zeroVal<typename AttributeType::ValueType>(), defaultValue);
+    appendAttribute<ValueType, CodecType>(tree, name, zeroVal<ValueType>(), stride, defaultValue);
```

Comment:
This method should fail if ValueType is Name due to zeroVal, perhaps a static_assert here to confirm this?